R/zzz.R

```diff
@@ -6,7 +6,10 @@
 
 .onLoad <- function(libname, pkgname) {
   make_flatline_reg()
-  make_quantile_reg()
-  make_smooth_quantile_reg()
+  if (!exists("quantile_reg")) {
+    make_quantile_reg()
+  }
+  if (!exists("smooth_quantile_reg")) {
+    make_smooth_quantile_reg()
+  }
```

Comment:
I haven't seen this in other packages that create parsnip models. This works? If yes, then we should delete the `.Rprofile` fix I added in #219

Reply:
I wish the `check_model_doesnt_exist` internal of parsnip were actually exposed for use, but this was working fine for me (I've been `load_all`'ing epipredict from other places, so I didn't notice the `.Rprofile`).